<ui version="4.0" >
 <author>Andrey Rijov</author>
 <class>Screenshot_Options_Window</class>
 <widget class="QDialog" name="Screenshot_Options_Window" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Screenshot Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="0" column="0" >
      <widget class="QLabel" name="Label__Screenshot_Save_Format" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Save Format:</string>
       </property>
       <property name="buddy" >
        <cstring>RB_Format_PPM</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QRadioButton" name="RB_Format_PPM" >
       <property name="text" >
        <string>Original (PPM)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QRadioButton" name="RB_Format_Jpeg" >
       <property name="text" >
        <string>Jpeg</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QRadioButton" name="RB_Format_PNG" >
       <property name="text" >
        <string>PNG</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <layout class="QGridLayout" name="gridLayout_3" >
     <item row="0" column="0" >
      <widget class="QLabel" name="Label_Jpeg_Quality" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Jpeg Quality:</string>
       </property>
       <property name="buddy" >
        <cstring>SB_Jpeg_Quality</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QSlider" name="HS_Jpeg_Quality" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimum" >
        <number>20</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="singleStep" >
        <number>5</number>
       </property>
       <property name="value" >
        <number>75</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QSpinBox" name="SB_Jpeg_Quality" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimum" >
        <number>20</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="singleStep" >
        <number>5</number>
       </property>
       <property name="value" >
        <number>75</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="Line" name="line_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" colspan="2" >
      <widget class="QCheckBox" name="CH_Screenshot_Folder" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Shared Folder to Save All Screenshots</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="Label_Folder_Path" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Folder Path:</string>
       </property>
       <property name="buddy" >
        <cstring>Edit_Screenshot_Folder</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="Edit_Screenshot_Folder" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QToolButton" name="TB_Screenshot_Folder" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Button_OK" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Cancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>RB_Format_PPM</tabstop>
  <tabstop>RB_Format_Jpeg</tabstop>
  <tabstop>RB_Format_PNG</tabstop>
  <tabstop>HS_Jpeg_Quality</tabstop>
  <tabstop>SB_Jpeg_Quality</tabstop>
  <tabstop>CH_Screenshot_Folder</tabstop>
  <tabstop>Edit_Screenshot_Folder</tabstop>
  <tabstop>TB_Screenshot_Folder</tabstop>
  <tabstop>Button_OK</tabstop>
  <tabstop>Button_Cancel</tabstop>
 </tabstops>
 <includes/>
 <resources/>
 <connections>
  <connection>
   <sender>RB_Format_Jpeg</sender>
   <signal>toggled(bool)</signal>
   <receiver>HS_Jpeg_Quality</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>154</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel" >
     <x>291</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_Format_Jpeg</sender>
   <signal>toggled(bool)</signal>
   <receiver>SB_Jpeg_Quality</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>165</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel" >
     <x>367</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_Format_Jpeg</sender>
   <signal>toggled(bool)</signal>
   <receiver>Label_Jpeg_Quality</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>156</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>291</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_Format_PPM</sender>
   <signal>toggled(bool)</signal>
   <receiver>Label_Jpeg_Quality</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>54</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel" >
     <x>265</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_Format_PPM</sender>
   <signal>toggled(bool)</signal>
   <receiver>HS_Jpeg_Quality</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>82</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>316</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_Format_PPM</sender>
   <signal>toggled(bool)</signal>
   <receiver>SB_Jpeg_Quality</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>28</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel" >
     <x>380</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_Format_PNG</sender>
   <signal>toggled(bool)</signal>
   <receiver>Label_Jpeg_Quality</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>191</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel" >
     <x>270</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_Format_PNG</sender>
   <signal>toggled(bool)</signal>
   <receiver>HS_Jpeg_Quality</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>215</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel" >
     <x>274</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RB_Format_PNG</sender>
   <signal>toggled(bool)</signal>
   <receiver>SB_Jpeg_Quality</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>219</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel" >
     <x>372</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Screenshot_Folder</sender>
   <signal>toggled(bool)</signal>
   <receiver>Label_Folder_Path</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>84</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel" >
     <x>70</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Screenshot_Folder</sender>
   <signal>toggled(bool)</signal>
   <receiver>Edit_Screenshot_Folder</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>135</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel" >
     <x>156</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Screenshot_Folder</sender>
   <signal>toggled(bool)</signal>
   <receiver>TB_Screenshot_Folder</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>241</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel" >
     <x>374</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Screenshot_Options_Window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>349</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel" >
     <x>74</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HS_Jpeg_Quality</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SB_Jpeg_Quality</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>291</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel" >
     <x>360</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SB_Jpeg_Quality</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HS_Jpeg_Quality</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>360</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel" >
     <x>291</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
